### DISCUSSION TITLE
I couldn't build and run the project locally

### DISCUSSION THREAD

**Discussion by Hamzablm** [provectus/kafka-ui#2178] [Q&A] [answered]
I want to build kafka-ui locally and run it but I got compilation errors:
<img width="627" alt="image" src="https://user-images.githubusercontent.com/32052228/174167109-68a70a9a-03af-485b-846e-6a1ce834eb54.png">

Note that I couldn't find the DTOs that are generating the compilation error in the project:
<img width="681" alt="image" src="https://user-images.githubusercontent.com/32052228/174167900-54117b74-ebfb-4f88-8928-80970971c5d3.png">

Any help would be appreciated! Thanks!

**Comment by Haarolean** [comment, marked as answer]
Hey, thanks for the interest in the project.
You have to build the app via maven's goals "clean package" with `generate-spring-webflux-api` profile (which is enabled by default).
